java/component/keyple-plugin/remotese/src/main/java/org/eclipse/keyple/plugin/remotese/pluginse/VirtualObservableReaderImpl.java

```diff
@@ -116,18 +115,12 @@ public final void clearObservers() {
 
     @Override
     public final void notifySeProcessed() {
-        if (forceClosing) {
-            try {
-                // close the physical channel thanks to CLOSE_AFTER flag
-                processSeRequest(null, CLOSE_AFTER);
-                logger.trace(
-                        "Explicit communication closing requested, starting removal sequence.");
-            } catch (KeypleReaderException e) {
-                logger.error("KeypleReaderException while terminating. {}", e.getMessage());
-            }
-        } else {
-            logger.trace("Explicit physical channel closing already requested.");
+        if (logger.isTraceEnabled()) {
```

Comment:
Check this please

Reply:
Olivier, I let you check this method

Reply:
From what I understand, the method should remains the same